utils/generate_api.py

```diff
@@ -590,9 +590,15 @@ def read_modules() -> Any:
             parts_new = {}
 
             for m in params:  # pylint: disable=invalid-name
-                a = dict(  # pylint: disable=invalid-name
-                    type=m["schema"]["type"], description=m["description"]
-                )  # pylint: disable=invalid-name
+                if "description" in m:
+                    a = dict(  # pylint: disable=invalid-name
+                        type=m["schema"]["type"], description=m["description"]
+                    )  # pylint: disable=invalid-name
+                else:
+                    a = dict(  # pylint: disable=invalid-name
+                        type=m["schema"]["type"]
+                    )  # pylint: disable=invalid-name
```

Comment:
Make it the same as below for `default`, create `a` (really could use a better variable name) first, then `update` it with `description` if any,